docs/prover/changelog/prover_changelog.md

```diff
@@ -5,6 +5,47 @@ Prover Release Notes
 ```{contents}
 ```
 
+5.0.5 (November 21, 2023)
+-------------------------
+### CVL
+- [feat] Allowing calling Solidity functions from within CVL hooks
+- [feat] {ref}`direct-storage-access`
+- [feat] Support for exhaustive parametric methods. Now `method f` calls will check for all methods in all contracts in Scene. The set of checked contracts can be limited with `--parametric_contracts Contract1 Contract2`
```

Comment:
I would probably add a link to the changes guide here.

Reply:
I added a header instead